setup.py

```diff
@@ -1,113 +1,6 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
+# This file is only needed for editable installs (using pyproject.toml)
```

Comment:
just a bare stub file now, needed only for editable installs until  https://github.com/pypa/setuptools/issues/2816